Matthias/sql 99 fonts search update

msbt opened the pull request:
## Summary of the changes / Why this is an improvement

This PR features 
- the new styling for the (now dedicated) SQL-99 docs, also reverted search for this repo to use the legacy/internal one
- fixed mobile view on ``search.html``
- changed primary font to ``Inter`` and secondary to ``Poppins``, removed Blender
- small fix to prevent console errors when no navbar is available

This could replace https://github.com/crate/sql-99/pull/77

jayeff reviewed (approved):
My personal preference for sql-99 specific code would be to try to keep as much as possible within the sql-99 repo for improved separation of concerns, easier maintainability, etc

Imagine a scenario with a minor CSS fix for sql-99 html-element could be a small fix in sql-99 repo + building docs vs with this approach requires change in docs-theme, releasing new version, bumping version of theme in sql-99 repo + building docs. Optimally all other projects based on the theme should be bumped as well.

I'm not sure how feasible it would be to implement this in the current theme though and AFAIK and overhaul of crate-docs-theme is in the pipeline already. For this rework these requirements could potentially be considered.

msbt commented:
> My personal preference for sql-99 specific code would be to try to keep as much as possible within the sql-99 repo for improved separation of concerns, easier maintainability, etc

I'm with you on that one, but removing/changing the whole navigation blocks for header and footer and customizing other parts of the theme can't be done from within the sql-99 repo, at least I wouldn't know how.

> I'm not sure how feasible it would be to implement this in the current theme though and AFAIK and overhaul of crate-docs-theme is in the pipeline already. For this rework these requirements could potentially be considered.

Also my thoughts, make this a quick fix and include a proper plan when we're going to revamp the theme

msbt commented on src/crate/theme/rtd/crate/layout.html:
```diff
@@ -69,6 +100,9 @@
             {% block body %}{% endblock %}
           </div>
+
+          {% if project != 'SQL 99' %}
```
right, I thought that was a global setting, I'll adjust accordingly

msbt replied:
done @jayeff

msbt commented on src/crate/theme/rtd/crate/layout.html:
```diff
@@ -195,7 +231,7 @@ <h2>Feedback</h2>
   {% endif %}
 
-
+  {% if project != 'SQL 99' %}
```
Thanks, will look into it!

msbt replied:
done @jayeff

jayeff commented:
> @jayeff I'm with you on that one, but removing/changing the whole navigation blocks for header and footer and customizing other parts of the theme can't be done from within the sql-99 repo, at least I wouldn't know how.

Without checking feasibility of this approach what comes to mind is some kind of hook to override certain blocks in the separate projects with a fallback/default block in the theme for projects which do not require special behaviour.

> Also my thoughts, make this a quick fix and include a proper plan when we're going to revamp the theme

👍

msbt commented:
@jayeff @amotl I think all the changes/bugfixes/reverts are implemented now, can you have another look?

Changelog so far:
- Fixed mobile view on ``search.html``
- Added new look for SQL-99 repository (brought back legacy search)
- Changed primary font to ``Inter`` and secondary to ``Poppins``, removed Blender
- Small fix to prevent console errors when no promo/navbar is available
- hide GitHub feedback-box on SQL-99 repo
- Fix for regression: Had to bring back `pygments.css` in `base.html`
- hide injected footer from RTD

jayeff reviewed (approved):
Did just a quick sweep over the PR as I reviewed it earlier already.

LGTM

amotl reviewed (approved):
Thank you very much! I've just rebased your branch on top of the main branch.
